sdk/authorization/arm-authorization-profile-2019-03-01-hybrid/src/authorizationManagementClientContext.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+import * as Models from "./models";
+import * as msRest from "@azure/ms-rest-js";
+import * as msRestAzure from "@azure/ms-rest-azure-js";
+
+const packageName = "@azure/arm-authorization-profile-2019-03-01-hybrid";
+const packageVersion = "1.0.0";
+
+export class AuthorizationManagementClientContext extends msRestAzure.AzureServiceClient {
+  credentials: msRest.ServiceClientCredentials;
+  subscriptionId: string;
+  apiVersion?: string;
+
+  /**
+   * Initializes a new instance of the AuthorizationManagementClient class.
+   * @param credentials Credentials needed for the client to connect to Azure.
+   * @param subscriptionId The ID of the target subscription.
+   * @param [options] The parameter options
+   */
+  constructor(credentials: msRest.ServiceClientCredentials, subscriptionId: string, options?: Models.AuthorizationManagementClientOptions) {
+    if (credentials == undefined) {
+      throw new Error('\'credentials\' cannot be null.');
+    }
+    if (subscriptionId == undefined) {
+      throw new Error('\'subscriptionId\' cannot be null.');
+    }
+
+    if (!options) {
+      options = {};
+    }
+    if(!options.userAgent) {
+      const defaultUserAgent = msRestAzure.getDefaultUserAgentValue();
+      options.userAgent = `${packageName}/${packageVersion} ${defaultUserAgent}`;
+    }
+
+    super(credentials, options);
+
+    this.apiVersion = '2015-07-01';
```

Comment:
The API version is that old?

Reply:
Apparently yes. Here is the link to the profile https://github.com/Azure/azure-rest-api-specs/blob/master/profile/2019-03-01-hybrid.json#L24